lib/presentation/mixins/chat_details_tab_mixin.dart

```diff
@@ -59,6 +61,8 @@ mixin ChatDetailsTabMixin<T extends StatefulWidget>
   static const _mediaFetchLimit = 20;
   static const _linksFetchLimit = 20;
   static const _filesFetchLimit = 20;
+  static const _maxMembers = 30;
```

Comment:
not get this idea? what is it?

Reply:
I wrote ADR for this ticket, pls check it more details related to `_maxMembers` variable